src/site/xdoc/manual/configuration.xml.vm

```diff
@@ -2146,6 +2146,32 @@ public class AwesomeTest {
       before falling back to the default class loader.
     </td>
   </tr>
+  <tr>
+    <td><a name="allowedLdapClasses"/>log4j2.allowedLdapClasses</td>
+    <td>LOG4J_ALLOWED_LDAP_CLASSES</td>
+    <td>&nbsp;</td>
+    <td>
+      System property that specifies fully qualified class names that may be accessed by LDAP. The classes
+      must implement Serializable. By default only Java primative classes are allowed.
```

Comment:
primitive vs primative

Reply:
> primitive vs primative

Noted, thank you @axdimitar! Will fix this soon.